lib/OfxParser/Ofx.php

```diff
@@ -210,4 +210,43 @@ private function createDateTimeFromStr($dateString, $ignoreErrors = false)
         throw new \Exception("Failed to initialize DateTime for string: " . $dateString);
     }
 
+    /**
+     * Create a formated number in Float according to different locale options
+     *
+     * Supports:
+     * 000,00 and -000,00
+     * 0.000,00 and -0.000,00
+     * 0,000.00 and -0,000.00
+     * 000.00 and 000.00
+     *
+     * @param  string  $amountString
+     * @return \Float | $amountString
```

Comment:
Sorry to be stickler, but you change to just
```php
@return float
```
As `\Float` implies a class, I don't think we need the `| $amountString` either